examples/connect-js/README.md

```diff
@@ -1,7 +1,17 @@
 # Example "connect-js"
 
-[![GitPod Logo](../../doc/run-in-gitpod.png)](https://gitpod.io/#example=bind-one-way/https://github.com/eclipsesource/tabris-decorators/tree/master/examples/connect)
+[![GitPod Logo](../../doc/run-in-gitpod.png)](https://gitpod.io/#example=bind-one-way/https://github.com/eclipsesource/tabris-decorators/tree/master/examples/connect-js)
 
 ## Description
 
-TODOC
+Note: A TypeScript/JSX variant of the this plain JavaScript example can be found [here](../connect). A JavaScript/JSX variant can be found [here](../connect-jsx).
+
+Demonstrates the use of the `connect` decorator function to connect custom components to a redux store. This app sets up a redux store with a root state object containing a string  and a number property `myString` and `myNumber`. Two actions are supported by that store: `SET_RANDOM_NUMBER` sets `myNumber` to a random number, and `TOGGLE_STRING` sets `myString` to `'HelloWorld'` or `'Another Hello World'` depending on the actions' `checked` boolean property.
+
+The file `types.d.ts` makes this configuration known globally for IDEs that support JavaScript projects with TypeScript declarations, i.e. Visual Studio Code. It also defines some type aliases that can be used in JsDoc annotations. The file has no impact at runtime.
+
+The app then creates instances of two components connected to the store:
+
+The `ExampleComponent` is a custom component with a `text` property and a `toggle` event. The `text` property controls what text the component displays and is mapped to the `myString` property of the root state object. The `toggle` event can be triggered by the components `CheckBox` and is mapped to the `TOGGLE_STRING` action. Therefore checking/unchecking the check box toggles the message.
+
+The `FunctionalComponent` is a functional component that produces a `Composite` that wraps a single `Button` widget. (It could also directly return a button, it is done this way to showcase the usage of the `apply` feature of `connect`.) The buttons `text` property is mapped to the `myNumber` property of the root state object. The buttons select event is mapped to the `SET_RANDOM_NUMBER` action. Therefore tapping the button changes its text.
```

Comment:
> The buttons `text` property

should be

> The button's `text` property

Reply:
> The buttons select event

should be

> The button's select event

Reply:
I guess it is better to write one sentence per line.

Markdown will format this as floating text. There is only a new paragraph after two line breaks.